docs/reference/ml/ml-shared.asciidoc

```diff
@@ -688,6 +693,12 @@ tag::groups[]
 A list of job groups. A job can belong to no groups or many.
 end::groups[]
 
+tag::include-model-definition[]
+Specifies if the model definition should be returned in the response. Defaults 
+to `false`. When `true`, only a single model must match the ID patterns 
+provided, otherwise a bad request is returned.
```

Comment:
> otherwise a bad request is returned.

It seems like this is referring to a 400 response code, so perhaps this circumstance should be added to the "response codes" section.